packages/electrumAnalyzer/electrumAnalyzer.0.3.5-2/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: "david.chemouil+electrum@onera.fr"
+authors: ["David Chemouil" "Julien Brunel" "Denis Kuperberg"]
+homepage: "https://forge.onera.fr/projects/electrum"
+bug-reports: "https://forge.onera.fr/projects/electrum"
+license: "GPL-3+"
+tags: "org:onera"
+dev-repo: "https://github.com/grayswandyr/electrumAnalyzer.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+remove: [ [  "ocaml" "%{etc}%/electrumAnalyzer/_oasis_remove_.ml" "%{etc}%/electrumAnalyzer" ]
+]
+depends: [
+  "batteries" {build}
+  "cmdliner" {build}
+  "menhir" {build & >= "20150720"}
+  "ocamlfind" {build}
+  "pprint" {build}
```

Comment:
there is a virtual `ocamlbuild` package for 4.02. It is just unusual to have no build dependencies but many build dependencies. I've just commented on that because that's the type of general mistakes that packagers are doing and that we are used to double-check before merging.